src/Storages/Distributed/DistributedAsyncInsertDirectoryQueue.h

```diff
@@ -100,9 +100,9 @@ class DistributedAsyncInsertDirectoryQueue
 
     void addFile(const std::string & file_path);
     void initializeFilesFromDisk();
-    void processFiles(const SettingsChanges & settings_changes = {});
+    void processFiles(const SettingsChanges & settings_changes = {}, bool force = false);
```

Comment:
I see `processFiles` is called in 2 places, so may be remove the default value and make it obligatory to manually be set?